source/frontends/libretro/serialisation.cpp

```diff
@@ -25,13 +29,32 @@ namespace
 
   AutoFile::AutoFile()
   {
+#ifdef _WIN32
+    CHAR szTempFileName[MAX_PATH];
+    CHAR lpTempPathBuffer[MAX_PATH];
+
+    DWORD dwRetVal = GetTempPathA(MAX_PATH, lpTempPathBuffer);
+    if (dwRetVal > MAX_PATH || (dwRetVal == 0))
+    {
+        throw std::runtime_error("Cannot create temporary file");
```

Comment:
overall indentation is unfortunately non uniform, but each file's is.

Reply:
It is time to force a decent formatting standards. I will resuscitate an old attempt https://github.com/AppleWin/AppleWin/issues/1202

Reply:
fixed.

i noticed the inconsistency, and did my best to maintain it on a per-file basis.